include/world_builder/world.h

```diff
@@ -188,6 +191,14 @@ namespace WorldBuilder
                                   const double depth,
                                   const unsigned int composition_number,
                                   size_t number_of_grains) const;
+      /**
+       * Returns a PlaneDistances object that has the distance from and along a feature plane,
+       * caculated from the coordinates and the depth of the point.
```

Comment:
please document the paramters here, especially that the point is in cartesian coordinates. You can use `@param point desciption` or `\param point description`.